russstation/code/modules/mob/living/carbon/human/species_types/dwarf.dm

```diff
@@ -24,13 +24,15 @@
 	var/dwarf_hair = pick("Beard (Dwarf)", "Beard (Very Long)", "Beard (Full)")
 	C.facial_hairstyle = dwarf_hair
 	C.update_hair()
+	// dwarves can see ghosts! and no putting them to rest with slabs, that would be too much
+	C.AddComponent(/datum/component/spookable)
```

Comment:
Less of a technical issue and more of a gameplay concern: I'm not sure why dwarfs need to be able to see ghosts, and it could lead to some meta info being shared around. Although since it's non-technical and not an issue in most cases, it's probably fine it leave this be and then change it later if it turns out people are abusing it.

Reply:
If it does lead to meta ill remove it

Reply:
in dwarf fortress, when your dwarves die they need to be put to rest properly or they haunt the fortress and distress the living dwarves. i thought it would be a cute addition that gives ghost players a little more spice. cool with it not being included if necessary